client/src/Components/Pages/LandingPage/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './style.css'
+import HeroComp from './HeroSection'
+
+
+class LandingPage extends React.Component  {
+
+   render(){
+    return (    
+       <div > 
+           <HeroComp history={this.props.history} />
```

Comment:
I think you can use react-router's `withRouter` higher-order component to get access to history in HeroComp without having to pass it down using props.  See here:
https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/withRouter.md

Basically, in HeroComp, if you export using withRouter() (so `export default withRouter(HeroComp)`), then you get access to this.props.history automatically.